### PR TITLE
adds video store

### PR DESCRIPTION
**Pull request by audreyandoy**
Added intro and submission pages

### CODE REVIEW

File: video-store/part-1-submit.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+##### !end-question
+
+##### !placeholder
+The URL to your project repo on GitHub: https://github.com/<your-username>/<project-name>
+##### !end-placeholder
+
+##### !answer
+
+/.+/
```
Comment (CheezItMan) [inline video-store/part-1-submit.md:24]: Just a suggestion to help make sure it's a valid PR.

```suggestion
/^https:\/\/github\.com\/[^\/]+\/[^\/]+\/pull\/\d?\d[\/]?/?/
```

File: video-store/part-2-submit.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+##### !end-question
+
+##### !placeholder
+The URL to your project repo on GitHub: https://github.com/<your-username>/<project-name>
+##### !end-placeholder
+
+##### !answer
+
+/.+/
```
Comment (CheezItMan) [inline video-store/part-2-submit.md:24]:
```suggestion
/^https:\/\/github\.com\/[^\/]+\/[^\/]+\/pull\/\d?\d[\/]?/?/
```

**Review by CheezItMan — APPROVED** [review]
:shipit:  LGTM

File: video-store/part-1-submit.md (hunk 1 above)
Comment (audreyandoy) [reply video-store/part-1-submit.md:24]: Thank you!!!

**Pull request merged**